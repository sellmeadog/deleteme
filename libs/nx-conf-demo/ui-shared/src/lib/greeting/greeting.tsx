/* eslint-disable jsx-a11y/accessible-emoji */
import React, { FC } from 'react';

import { View, Text, StyleProp, TextStyle, ViewStyle } from 'react-native';

export interface GreetingProps {
  containerStyle?: StyleProp<ViewStyle>;
  textStyle?: StyleProp<TextStyle>;
}

export const Greeting: FC<GreetingProps> = ({ containerStyle, textStyle }) => {
  return (
    <View style={containerStyle}>
      <Text accessibilityRole="header" style={textStyle}>
        Welcome to Nx Conf ðŸ‘‹
      </Text>
    </View>
  );
};
