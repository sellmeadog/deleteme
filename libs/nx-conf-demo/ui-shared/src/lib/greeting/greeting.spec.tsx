import React from 'react';
import { render } from '@testing-library/react-native';

import { Greeting } from './greeting';

describe('Greeting', () => {
  it('should render successfully', () => {
    const { toJSON } = render(<Greeting />);
    expect(toJSON()).toMatchSnapshot();
  });
});
