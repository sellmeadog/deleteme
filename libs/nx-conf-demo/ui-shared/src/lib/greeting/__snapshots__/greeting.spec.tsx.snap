// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting should render successfully 1`] = `
<View>
  <Text>
    Welcome to Nx Conf ðŸ‘‹
  </Text>
</View>
`;
